doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -664,7 +617,7 @@ Passthrough a Hard Disk to RTVM
 
       passthru_vpid=(
       ["eth"]="8086 156f"
```

Comment:
Is this one correct? I'm wondering especially in light with the paragraph above that states the Ethernet adapter is not recognized.

Reply:
No, I updated all PCI device IDs and busses that's listed at the end of this document just now.